### PR TITLE
New example - multiple image classification

### PR DESCRIPTION
**Pull request by handav**
There could still be more work done on this - i.e. saving/rewriting to the json at multiple points, potentially a drag-and-drop option to upload/process images - but I think it would be good to commit it so far since it works? Totally open to feedback if everyone thinks otherwise!

### CODE REVIEW

File: p5js/00_ImageClassification_MultipleImages/assets/data.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"images": ["image_0004.jpg", "image_0005.jpg", "image_0006.jpg", "image_0007.jpg", "image_0008.jpg", "image_0009.jpg", "image_00092.jpg", "image_0010.jpg", "image_00102.jpg", "image_00103.jpg", "image_0011.jpg", "image_00112.jpg", "image_0012.jpg", "image_00122.jpg", "image_0013.jpg"]}
```
Comment (shiffman) [inline p5js/00_ImageClassification_MultipleImages/assets/data.json]: This is probably silly, but I wonder if the python script could "prettify" the JSON so it looks more like. the following?

```json
{
   "images":[
      "image_0004.jpg",
      "image_0005.jpg",
      "image_0006.jpg",
      "image_0007.jpg",
      "image_0008.jpg",
      "image_0009.jpg",
      "image_00092.jpg",
      "image_0010.jpg",
      "image_00102.jpg",
      "image_00103.jpg",
      "image_0011.jpg",
      "image_00112.jpg",
      "image_0012.jpg",
      "image_00122.jpg",
      "image_0013.jpg"
   ]
}
```

File: p5js/00_ImageClassification_MultipleImages/sketch.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+let img;
+current_index = 0;
+all_images = []
+display = true;
+display_time = 750;
+predictions = []
+
+function append_images(){
+    for (var i in data.images){
```
Comment (shiffman) [inline p5js/00_ImageClassification_MultipleImages/sketch.js]: This sort of loop is a little less familiar to beginners. I would suggest a `for` loop.

File: p5js/00_ImageClassification_MultipleImages/sketch.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        img_path = data.images[i];
+        all_images.push(get_image_path(img_path));
+    }
+}
+
+function preload(){
+    data = loadJSON('/assets/data.json', append_images);
```
Comment (shiffman) [inline p5js/00_ImageClassification_MultipleImages/sketch.js]: If `loadJSON()` is used in `preload()` then there's no need for a callback. You can just call `append_images()` in `setup()`.

**Comment by handav** [comment]
I fixed everything except for prettifying the JSON! I'll still look into that.

File: p5js/00_ImageClassification_MultipleImages/sketch.js [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+ML5 Example
+00_ImageNet_Simple
+Simple Image Classification using p5.js
+=== */
+
+// Initialize the ImageNet method with the MobileNet model.
+const classifier = new ml5.ImageClassifier('MobileNet');
+
+let img;
+currentIndex = 0;
```
Comment (shiffman) [inline p5js/00_ImageClassification_MultipleImages/sketch.js]: Sorry, noticed one more thing! These should probably all have a `let` and the two `[]` lines should have a semi-colon. This is so great though -- I'll merge and we can do a little cleanup after if need be!

**Comment by shiffman** [comment]
Woohoo! 🎉

File: p5js/00_ImageClassification_MultipleImages/sketch.js (hunk 4 above)
Comment (handav) [reply p5js/00_ImageClassification_MultipleImages/sketch.js]: Thank you so much! I'll start using that format going forward.

**Pull request merged**